### ISSUE TITLE
./mitm.sh: line 166: arpoison: command not found

### ISSUE THREAD

**Issue by tai-euler** [phackt/mitm#1]
root@lenovo-PC:~/mitm# sudo ./mitm.sh -g -n -x -i wlan0 192.168.1.1 192.168.1.11
INTERACTIVE_MODE On
HTTP_INTERCEPTION On
HTTPS_STRIPPING On
Flushing iptables...
**./mitm.sh: line 166: arpoison: command not found**

**why do I get this error on kali 2017?**

**Comment by tai-euler** [comment]
**this is in the source code:**

#####################################
# ARP poisoning
#####################################

arpoison ${GATEWAY} ${TARGET}

#####################################


**is there something wrong?**

**Comment by phackt** [comment]
@tai-lopez Thanks to notify that, i updated the PATH vars in the script can you let me know if everything is working on your side right now ? (and close the case is OK).

Thks

**Comment by tai-euler** [comment]
nice thanks! now i got other  errors.
I had to copy arpoison.sh into my /root/bin directory to work properly.
after that I get another error, I could not solve that.

**xterm: can't execvp root/mitmproxy/mitmproxy: Permission denied**

I copied mitmproxy outside mitm folder to root/ and made another copy inside so we have mitmproxy/mitmproxy and still nothing.
Than i tried chmod +x mitmproxy.tar in both folders that maybe helps but still the same error.
I run mitms.sh always with sudo command and still nothing.
